src/python/grpcio/grpc/_cython/_cygrpc/aio/server.pxd.pxi

```diff
@@ -41,6 +41,18 @@ cdef class RPCState(GrpcCallWrapper):
     cdef Operation create_send_initial_metadata_op_if_not_sent(self)
 
 
+cdef class _ServicerContext:
+    cdef RPCState _rpc_state
+    cdef object _loop
```

Comment:
Nit: Any chance we can get type annotations in the comments?

Reply:
Type annotation added, although the best I can do is fake type annotations in comment.